### DISCUSSION TITLE
Recording Web UI (PC & Mobile)

### DISCUSSION THREAD

**Discussion by Deishelon** [blakeblackshear/frigate#6869] [General]
Hi,

Are there any UI/UX improvement planned for 'Recording' page on PC/Mobile?

For me currently, the 'Recording' page on PC is rendered over the height of the window, so the video player & timeline UI do not fit into the windows's height, which is quite annoying to scroll, and hard to view the recording as video player is bigger than the page height. 

I had to blur the recording, but you can see the scroll bar on the right.
![Screenshot_20230621_113242](https://github.com/blakeblackshear/frigate/assets/11800090/13fd382e-dd61-4896-b5fc-c956a24add5a)

On the phone, on another hand, the height is ok, but it feels like there is a lot of wasted space by the UI which goes over the video clip, but we have a lot of empty space below the video player. So the control UI could live there?

![photo_4940908731139468105_y](https://github.com/blakeblackshear/frigate/assets/11800090/fae17384-63d7-4b76-b54d-dfb889e219f5)

**Comment by NickM-27** [comment]
Yes, there are many plans for recordings in future releases after 0.13

**Reply by Deishelon** [reply]
can you point into any current GH issues to follow about this ? where can I view milestones for future releases?

**Reply by NickM-27** [reply]
You can see the items for the current in progress release here https://github.com/blakeblackshear/frigate/pull/6262
